packages/apps/plugins/plugin-markdown/src/MarkdownPlugin.tsx

```diff
@@ -54,21 +69,25 @@ export const MarkdownPlugin = (): PluginDefinition<MarkdownPluginProvides> => {
     data: { composer: ComposerModel; properties: MarkdownProperties };
     role?: string;
   }) => {
+    const onChange: NonNullable<ComposerOptions['onChange']> = useCallback(
+      (content) => state.onChange.forEach((onChange) => onChange(content)),
+      [state.onChange],
+    );
+
     return (
       <EditorMain
         model={composer}
         properties={properties}
         layout='standalone'
-        onChange={(text) => state.onChange.forEach((onChange) => onChange(text))}
+        options={{ onChange, editorMode: settings.values.editorMode }}
```

Comment:
Would prefer these were adjacent instead of nested so React isn’t diffing props that use an object literal.

Reply:
_Probably_ not a problem since the editor is so careful about when it remounts, but better not to have to think about it IMO.

Reply:
> Would prefer these were adjacent instead of nested so React isn’t diffing props that use an object literal.

Probably wise :+1: